### PR TITLE
fix: use latest webln if available in globalThis after init

### PR DESCRIPTION
**Pull request by im-adithya**
Title

### CODE REVIEW

File: src/lightning-address.ts [hunk 1]
```diff
@@ -233,12 +237,13 @@ export default class LightningAddress {
     options: ZapOptions = {},
   ): Promise<SendPaymentResponse> {
     const invoice = this.zapInvoice(args, options);
-    if (!this.webln) {
+    const webln = this.getWebLN()
+    if (!webln) {
```
Comment (rolznz) [inline src/lightning-address.ts:241]: can you also add this check in the boost function above? right now it doesn't check if webln is unavailable.

Also the `// mainly for TS` comment I think can be removed - it's not for TS, it is actually used to give a helpful error if webln does not exist

**Comment by bumi** [comment]
hmm... should we use the global webln? or shouldn't webln be passed in?

**Comment by rolznz** [comment]
> hmm... should we use the global webln? or shouldn't webln be passed in?

I think it is good to fallback to the global webln if you don't pass one in?

File: src/lightning-address.ts (hunk 1 above)
Comment (im-adithya) [reply src/lightning-address.ts:243]: Done!

**Review by rolznz — APPROVED** [review]
utACK

**Pull request merged**